src/schemas.ts

```diff
@@ -93,6 +93,7 @@ export interface SchemaFavourite extends SchemaRecord {
 export interface SchemaSocialChatGroup extends SchemaRecord {
     newestId?: string
     syncData?: string
+    type?: string
```

Comment:
Can you explain this?

Reply:
> Can you explain this?

Just tried to represent DM, private, or public. But not used. I will remove `type`